src/components/Card/Card.jsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import propTypes from "prop-types";
+import classNames from "classnames";
+import styles from "./Card.module.scss";
+import { Text } from "../Text/Text";
+
+export function Card({
+  disabled = false,
+  primaryText = "",
+  secondaryText = "",
+  className = "",
+  rightElement = "",
+  imageUrl = "",
+  ...props
+}) {
+  return (
+    <div
+      className={classNames(styles.card, className, {
+        [styles.card_disabled]: disabled,
+      })}
+      {...props}
+    >
+      {imageUrl && (
+        <div className={classNames(styles["card__image-container"])}>
+          <img src={imageUrl} alt="" />
+        </div>
+      )}
+      <div className={classNames(styles.card__text)}>
+        <Text size="h4">{primaryText}</Text>
+        <Text size="p1">{secondaryText}</Text>
```

Comment:
```suggestion
        <Text size="p">{secondaryText}</Text>
```

Reply:
I decided to not pass this prop at all in this case cuz default value is `p`